src/setup/scss/mixins/_font.scss

```diff
@@ -7,11 +11,11 @@
 //
 // returns  {font-size|line-height|font-weight}   Returns font-size and optional line-height and font-weight
 ////
-@mixin font($font-size: $_config--font-size, $line-height: null, $font-weight: null) {
-  @include font-size($font-size);
+@mixin font($font-size: config.$config--font-size, $line-height: null, $font-weight: null) {
+  @include font-size.font-size($font-size);
```

Comment:
should we use a custom namespace like e.g. `fontSizeMixins` to prevent the repetition of `font-size.font-size`?

Reply:
@janwidmer I would recommend to use a `mixin` namespace which wrapps all mixins if that is possible.

Reply:
simplified mixin usage within oder mixins